app/src/main/java/com/dreamisi/moviebase/FragmentMoviesDetails.kt

```diff
@@ -0,0 +1,16 @@
+package com.dreamisi.moviebase
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.fragment.app.Fragment
+
+class FragmentMoviesDetails: Fragment() {
```

Comment:
Самый актуальный подход к указанию разметки, это указывать разметку в конструкторе фрагмента
```suggestion
class FragmentMoviesDetails: Fragment(R.layout.fragment_movies_details)
```